packages/block-editor/src/components/font-appearance-control/index.js

```diff
@@ -5,6 +5,56 @@ import { CustomSelectControl } from '@wordpress/components';
 import { useMemo } from '@wordpress/element';
 import { __, sprintf } from '@wordpress/i18n';
 
+const FONT_STYLES = [
+	{
+		name: __( 'Regular' ),
+		value: 'normal',
+	},
+	{
+		name: __( 'Italic' ),
+		value: 'italic',
+	},
+];
+
+const FONT_WEIGHTS = [
```

Comment:
I know this wasn't introduced in this PR but wanted to confirm anyway.

I've cross-checked the names of the numbered values in the [standard](https://www.w3.org/TR/css-fonts-3/#font-weight-prop) with what we have here, and this is what I saw:

Value | Name in the standard      | Our current name
------|---------------------------|-----------------
100   | Thin                      | Ultralight
200   | Extra Light (Ultra Light) | Thin
300   | Light                     | Light
400   | Normal                    | Regular
500   | Medium                    | Medium
600   | Semi Bold (Demi Bold)     | Semibold
700   | Bold                      | Bold
800   | Extra Bold (Ultra Bold)   | Heavy
900   | Black (Heavy)             | Black

Wanted to check if this is fine or whether we want to align with the standard. These being values that core will provide, I'd lean to align their names with the standard. However, I understand that there may be a different "industry standard", so would like to hear what other people think of this. cc @kjellr @jasmussen

Reply:
Their names should definitely be industry standard, except for "Normal", which should be "Regular", or it will confuse the interface when selected.

Reply:
Yeah, we should be fine with the standard you noted here.